casa/OS/HostInfo.h

```diff
@@ -131,12 +131,22 @@ class HostInfo
     static ptrdiff_t swapFree();
     // </group>
 
+    // Introduce run-time memory limit re-specification capabilities
+    // <group>
+    static void setMemoryTotal(int memory);
```

Comment:
of course the input argument needs to be ptrdiff_t for the memory, how is that inconvenient?